testapp/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "testapp",
   "version": "0.0.1",
   "scripts": {
-    "start": "concurrently \"npm run tsc:w\" \"npm run lite\" ",
-    "tsc": "tsc",
-    "tsc:w": "tsc -w",
-    "lite": "lite-server"
+    "install_testapp": "npm install && npm run tsc",
+    "start": "npm run install_testapp && node scripts/web-server.js",
+    "start_detach": "node scripts/web-server.js &",
```

Comment:
Removing `install_testapp` from `start` in favor of `npm install`. Removing `start_detach` since its only used in Travis.